src/core_modules/capture-core/components/ListView/DownloadDialog/DownloadDialog.container.js

```diff
@@ -1,6 +1,6 @@
 // @flow
 import { connect } from 'react-redux';
-import DownloadDialog from './DownloadDialog.component';
+import { DownloadDialog as DownloadDialogComponent } from './DownloadDialog.component';
```

Comment:
NIT but this is the same as before. We had this conversation on some older PR saying that we could name the `DownloadDialog` directly as `DownloadDialogComponent` in the cases that we want to import this to a `.container`

Do you think we could apply this here?

Reply:
Removed in later PR.